### ISSUE TITLE
page_cache_length_unit always 'minutes'

### ISSUE THREAD

**Issue by msassmann** [tlovett1/simple-cache#97]
Hi!

When setting the the page cache length on admin page (advanced mode), 'page_cache_length_unit' is obviously always 'minutes'.
E.g. setting 11 hours, a cron with 11 minutes intervals is created.

Setting the option with 'sc_cache_purge_interval' filter works as expected:

```
function sc_custom_cache_purge_interval( $interval ) {
    return 11 * HOUR_IN_SECONDS;
}

add_filter( 'sc_cache_purge_interval', 'sc_custom_cache_purge_interval', 10, 1 );
```

**Comment by ram108** [comment]
@tlovett1 
Yes, I confirm the bug. Flush cache setting in admin panel does not work. The cache flushes after a few minutes. I wanted to open my own issue, but saw that it had already been open for a long time, and the bug was not fixed. In fact, the plugin is currently not performing its function.

@msassmann thanks for the manual fix.

**Comment by tlovett1** [comment]
This is fixed in 1.7.2
